stroyprombeton/views/catalog.py

```diff
@@ -70,8 +70,12 @@ def serialize_categories(categories):
     return response
 
 
+# @todo #556:120m  Show root categories as matrix.
```

Comment:
I believe 60m will be enough, but up to you

Reply:
@artemiy312 , if we have mockup, it's true. But here some UX thoughts would be realized